### PR TITLE
Vehicles - Configurable speed limiter step

### PR DESCRIPTION
**Pull request by veteran29**
**When merged this pull request will:**
- title

### CODE REVIEW

File: addons/vehicles/stringtable.xml [hunk 1]
```diff
@@ -148,5 +148,10 @@
             <Italian>Nasconde la voce Espelli dal menu azione. Richiede il riavvio del gioco.</Italian>
             <Czech>Schová akci vyskošení z vozidla z akčního menu. Vyžaduje restart hry.</Czech>
         </Key>
+        <Key ID="STR_ACE_Vehicles_SpeedLimiterStep">
+            <English>Speed limiter step</English>
```
Comment (mharis001) [inline addons/vehicles/stringtable.xml]:
```suggestion
            <English>Speed Limiter Step</English>
```
I think title case is better.

File: addons/vehicles/initSettings.sqf [hunk 2]
```diff
@@ -19,3 +19,12 @@
...
     true // needs restart
 ] call CBA_fnc_addSetting;
+
+[
+    QGVAR(speedLimiterStep),
+    "SLIDER",
+    LSTRING(SpeedLimiterStep),
+    ELSTRING(common,ACEKeybindCategoryVehicles),
+    [1, 10, 1, 0],
+    2
```
Comment (jonpas) [inline addons/vehicles/initSettings.sqf:29]: Feels like it should be default 5.

File: addons/vehicles/stringtable.xml [hunk 3]
```diff
@@ -149,7 +149,7 @@
             <Czech>Schová akci vyskošení z vozidla z akčního menu. Vyžaduje restart hry.</Czech>
         </Key>
         <Key ID="STR_ACE_Vehicles_SpeedLimiterStep">
-            <English>Speed limiter step</English>
+            <English>Speed Limiter Step</English>
             <German>Schrittgröße des Geschwindigkeitsbegrenzers</German>
             <Polish>Przeskok limitera prędkości</Polish>
```
Comment (Elgin675) [inline addons/vehicles/stringtable.xml:154]:
```suggestion
            <Polish>Przeskok limitera prędkości</Polish>
            <French>Pas du limiteur de vitesse</French>
```

File: addons/vehicles/initSettings.sqf (hunk 2 above)
Comment (veteran29) [reply addons/vehicles/initSettings.sqf:29]: Any other opinions?

Comment (bux) [reply addons/vehicles/initSettings.sqf:29]: 5

File: addons/vehicles/initSettings.sqf [hunk 4]
```diff
@@ -19,3 +19,12 @@
     },
     true // needs restart
 ] call CBA_fnc_addSetting;
+
+[
+    QGVAR(speedLimiterStep),
+    "SLIDER",
+    LSTRING(SpeedLimiterStep),
+    ELSTRING(common,ACEKeybindCategoryVehicles),
+    [1, 10, 1, 0],
```
Comment (veteran29) [inline addons/vehicles/initSettings.sqf:28]:
```suggestion
    [1, 10, 5, 0],
```

File: addons/vehicles/initSettings.sqf (hunk 2 above)
Comment (veteran29) [reply addons/vehicles/initSettings.sqf:29]: 5 it is.

File: addons/vehicles/initSettings.sqf [hunk 5]
```diff
@@ -25,6 +25,6 @@
     "SLIDER",
     LSTRING(SpeedLimiterStep),
     ELSTRING(common,ACEKeybindCategoryVehicles),
-    [1, 10, 1, 0],
+    [1, 10, 5, 0],
     2
```
Comment (jonpas) [inline addons/vehicles/initSettings.sqf:29]: Should we really use this? What if someone wants players to only be able to set speed limiter by 10ths or something. CBA wiki says it's not advisable to use as well.

Comment (PabstMirror) [reply addons/vehicles/initSettings.sqf:29]: -1 might make sense, would keep all speeds to whole numbers

File: addons/vehicles/initSettings.sqf [hunk 6]
```diff
@@ -19,3 +19,12 @@
...
     true // needs restart
 ] call CBA_fnc_addSetting;
+
+[
+    QGVAR(speedLimiterStep),
+    "SLIDER",
+    LSTRING(SpeedLimiterStep),
+    ELSTRING(common,ACEKeybindCategoryVehicles),
+    [1, 10, 5, 0],
+    2
```
Comment (veteran29) [inline addons/vehicles/initSettings.sqf:29]:
```suggestion
    [1, 10, 5, 0]
```

**Pull request merged**